integrated_channels/sap_success_factors/admin/__init__.py

```diff
@@ -32,11 +35,42 @@ class SAPSuccessFactorsEnterpriseCustomerConfigurationAdmin(admin.ModelAdmin):
     """
     Django admin model for SAPSuccessFactorsEnterpriseCustomerConfiguration.
     """
+
     list_display = (
         "enterprise_customer_uuid",
         "sapsf_base_url",
         "key",
+        "secret",
+        "sapsf_company_id",
+        "sapsf_user_id",
+        "sap_access_token",
+    )
+
+    readonly_fields = (
+        "sap_access_token",
     )
 
     class Meta(object):
         model = SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+    def sap_access_token(self, obj):
+        """
+        Return whether or not the configuration can provide access to SAP SuccessFactors.
+
+        Args:
+            obj: The instance of SAPSuccessFactorsEnterpriseCustomerConfiguration
+                being rendered with this admin form.
+        """
+        try:
+            (access_token, expires_at) = SAPSuccessFactorsAPIClient.get_oauth_access_token(
```

Comment:
Not necessary to wrap those items as a tuple; Python will unpack the two return values without that.